Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayForPod.xcodeproj/project.pbxproj

```diff
@@ -583,15 +591,14 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"iOS/Preview Content\"";
-				DEVELOPMENT_TEAM = JA85M8JD4Z;
 				ENABLE_PREVIEWS = YES;
 				INFOPLIST_FILE = iOS/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "testing-com.google.DaysUntilBirthday";
+				PRODUCT_BUNDLE_IDENTIFIER = com.google.DaysUntilBirthday;
```

Comment:
It looks like it doesn't have quotes on the [Main Branch](https://github.com/google/GoogleSignIn-iOS/blob/main/Samples/Swift/DaysUntilBirthday/DaysUntilBirthday.xcodeproj/project.pbxproj), but it probably doesn't make a difference if I add since `com.google.DaysUntilBirthdayUITests-iOS-` is in quotes. I can add if you like, let me know what you think.

Reply:
Ah no worries then!